docs/user/canonical-urls.rst

```diff
@@ -0,0 +1,53 @@
+Canonical URLs
+==============
+
+Canonical URLs allow you to have consistent page URLs across all your domains.
+This is mainly useful for search engines,
+so that they can send users to the correct version and domain of your documentation.
+
+The canonical URL is generated based on:
+
+* The default version of your project (usually "latest" or "stable").
+* The canonical :doc:`custom domain </custom-domains>` if you have one,
+  otherwise the default :ref:`subdomain <hosting:subdomain support>` will be used.
```

Comment:
This mentions how the canonical URL is generated but does not tell me what's the ending result in each case. I'm not sure to understand these bullets.